pkg/daemon/ceph/client/upgrade.go

```diff
@@ -70,10 +123,10 @@ func GetCephMonVersion(context *clusterd.Context) (*cephver.CephVersion, error)
 }
 
 // GetCephVersions reports the Ceph version of each daemon in the cluster
-func GetCephVersions(context *clusterd.Context) (*CephDaemonsVersions, error) {
-	output, err := getCephVersionsString(context)
+func GetCephVersions(context *clusterd.Context, clusterName string) (*CephDaemonsVersions, error) {
+	output, err := getCephVersionsString(context, clusterName)
 	if err != nil {
-		return nil, fmt.Errorf("failed to run ceph versions: %+v", err)
+		return nil, fmt.Errorf("failed to run ceph versions. %+v", err)
```

Comment:
This is adding the same error text as `getCephVersionsString`. Probably just `return err`

Reply:
bump